### DISCUSSION TITLE
Regarding the problem of importing .shp.zip data format

### DISCUSSION THREAD

**Discussion by xdxTao** [osm-search/Nominatim#3436] [Q&A]
Hi

I am using Nominatim4.4. I downloaded data in pbf format from geofabrik, but there are always some inaccuracies (different from the online API query provided by Nominatim).

I wonder if the errors will be improved if I change the data format to shp.zip instead of .pbf, but I downloaded the data in shp.zip format and don’t know how to import it into Nominatim after unzipping it.

I look forward to your help.

**Comment by mtmail** [comment]
Nominatim only understands *.osm, *.pdf data formats.

**Reply by xdxTao** [reply]
@mtmail   Yes, I know that.

But chatGPT told me that I can convert .shp.zip to pbf format. I tried to do this, and the data size became much smaller during the conversion, so I want to confirm whether this method is not feasible.

**Reply by mtmail** [reply]
chatGPT is wrong and wasted your time.

**Reply by xdxTao** [reply]
OK, got it.

I want to ask, if I want to import data in .shp.zip format to do address resolution and reverse address resolution, do you know any open source software as good as Nominatim that you can recommend? It needs to support http interface like Nominatim.

**Reply by mtmail** [reply]
I haven't seen a geocoding engine yet that imports *.shp files. The *.osm (XML) and *.pbf (binary, compressed, same data) contain more information.

https://wiki.openstreetmap.org/wiki/Geocoding lists other engines, they all have. a HTTP/REST API. As far as I know Pelias imports WOF (https://www.whosonfirst.org/) boundaries, those are not from OpenStreetMap. That could be an alternative. I don't know how to import just one country, you'd need to ask on their github issue tracker.
